### ISSUE TITLE
A Syloti Nagari font without the horaizontal line is needed.

### ISSUE THREAD

**Issue by Sagir8453** [googlei18n/noto-fonts#1400]
## Defect Report
 Syloti Nagari script is also written without the horizontal line.

### Title 
  > Should be brief but contain essential information.

### Font 
  NotoSansSylotiNagari-Regular.ttf

### Where the font came from, and when
  > For example:
  >   Site: https://noto-website-2.storage.googleapis.com/pkgs/NotoSansArmenian-hinted.zip
  >   Date: 2017-04-10 (preferred format)

### Font Version
 Current available on github, 4/12/2018

### Screenshot
  
![screenshot_2018-12-04-17-37-32-239_com android chrome](https://user-images.githubusercontent.com/22221772/49441514-bb35ae00-f7ec-11e8-9b00-03a22afda22b.png)
![screenshot_2018-12-04-17-40-35-297_com android chrome](https://user-images.githubusercontent.com/22221772/49441515-bbce4480-f7ec-11e8-9419-532399a45cb8.png)

**Comment by simoncozens** [comment]
I agree with this. The Unicode proposal supporting material [L2/388](https://www.unicode.org/L2/L2002/02388-syloti-proposal.pdf) also says:

>  The impression of words having a mātrā line is due to the medium, therefore, and not to the writing of the scribes themselves....  Syloti Nagri is closely related to Gujarati and Kaithi. Since these are not written with the mātrā line, it is not surprising that the same is true in Syloti Nagri manuscripts... Thus, we think the expectation that the mātrā line should be an integral component of modern Syloti Nagri type should be re-examined. The mātrā line is not a necessary element of Syloti Nagri type forms. It could be argued, in fact, that the mātrā line should be excluded from typefaces intended to closely match historic calligraphic traditions.

Noto's aim is to provide a "typeface for the nations" which covers all of the encoded Unicode codepoints. So the existing Syloti Nagari family is sufficient for this aim. But there are plans to produce "Noto Sans Hand", a more handwriting-focused font to be used for casual documents and handwriting primers.

My suggestion, then, is that Noto Sans Syloti Nagari maintains the headline, but when Noto Sans Hand Syloti Nagari is commissioned, this would not include a headline.
